### PR TITLE
Global style

### PR DESCRIPTION
**Pull request by Quentin-Macquart**
Hey Heloise, 
I add the globalStyle in all major parts. I did still have the problem transferring the dark mode to others files. I let you check before we have a debug session with Loic. I made all the correction the client asked for too. 

There are a plenty of files modified, but don't worry about it it's only conditions and css added. 
Happy reading!

### CODE REVIEW

File: src/pages/Food/BurgerMenu/index.jsx [hunk 1]
```diff
@@ -6,20 +6,12 @@ function BurgerMenu() {
...
-            <Link to="/Food">Menu du jour</Link>
-            <li>
-              <Link to="/Food/Profile">Profil alimentaire</Link>
-            </li>
-          </li>
-          <li>
-            <Link to="/Food/randomLunch">Random lunch</Link>
-          </li>
-        </ul>
+        <section>
```
Comment (HeloiseFerla) [inline src/pages/Food/BurgerMenu/index.jsx:9]: path in camelCase

File: src/pages/Office/index.jsx [hunk 2]
```diff
@@ -6,15 +6,17 @@ import 'swiper/swiper-bundle.min.css';
 import 'swiper/swiper.min.css';
 import 'swiper/components/navigation/navigation.min.css';
 import officeRoom from 'assets/Img/OfficePictures/officeRoom.png';
+import logo from 'assets/reservationBureau.png';
+import Header from '../../components/Header';
```
Comment (HeloiseFerla) [inline src/pages/Office/index.jsx]: components/ ...

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

File: src/pages/Food/BurgerMenu/index.jsx (hunk 1 above)
Comment (Quentin-Macquart) [reply src/pages/Food/BurgerMenu/index.jsx:9]: Mh...

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Food/FoodProfile/index.jsx [hunk 3]
```diff
@@ -88,9 +88,9 @@ function FoodProfile() {
       <BurgerMenu />
       <FoodProfileList>
         <div>
-          <h2>Votre profil alimentaire</h2>
+          <p className="votreProfil">Votre profil alimentaire</p>
```
Comment (Hacene010) [inline src/pages/Food/FoodProfile/index.jsx]: className in french ???

File: src/pages/Home/style.jsx [hunk 4]
```diff
@@ -89,20 +87,39 @@ const SHome = styled.section`
   }
 
   .meeting {
+    position: relative;
     display: flex;
```
Comment (Hacene010) [inline src/pages/Home/style.jsx:91]: With a display:flex; only, you don't need a flex-direction:row;, don't you ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Saw some evils :-)

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Food/FoodProfile/index.jsx (hunk 3 above)
Comment (Quentin-Macquart) [reply src/pages/Food/FoodProfile/index.jsx]: I promise i got the accent when i wrote it

**Review by Hacene010 — APPROVED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Pull request merged**